src/dynamodb.ts

```diff
@@ -0,0 +1,93 @@
+import { DynamoDB } from "aws-sdk";
+import { ClientConfiguration, ScanInput } from "aws-sdk/clients/dynamodb";
+import { DateTime } from "luxon";
+import { dynamoLogger } from "./handler";
+// @ts-ignore
+// tslint:disable-next-line:no-var-requires
+const AWSXRay = require("aws-xray-sdk");
+
+export class Dynamo {
+    private readonly config: ClientConfiguration;
+    private readonly tableName: string;
+    private readonly branch: string;
+    private readonly now: DateTime = DateTime.utc();
+    private readonly numOfScanners: number;
+
+    public constructor(config?: ClientConfiguration, branch?: string, numOfScanners: number = 4) {
+        this.config = config || { region: process.env.AWS_REGION || "eu-west-1" };
+        this.branch = branch || process.env.BRANCH || "local";
+        this.tableName = `cvs-${this.branch.toLowerCase()}-activities`;
+        this.numOfScanners = numOfScanners;
+    }
+
+    /**
+     * Returns the date formatted to CVS project standard.
+     * @param {DateTime} date A Luxon Datetime.
+     * @returns {string}
+     */
+    public static async toCVSDate(date: DateTime): Promise<string> {
+        return await date.toISO({ includeOffset: false }) + "Z";
```

Comment:
is luxon/DateTime asynchronous? Suspect it's not from my reading of the docs, so no await required.

Reply:
My mistake